app/src/org/commcare/views/widgets/WidgetFactory.java

```diff
@@ -61,7 +61,11 @@ public QuestionWidget createWidgetFromPrompt(FormEntryPrompt fep, Context contex
                             questionWidget = new EthiopianDateWidget(context, fep);
                         } else if (appearance != null && appearance.toLowerCase().equals("nepali")) {
                             questionWidget = new NepaliDateWidget(context, fep);
-                        } else {
+                        }else if(appearance != null && appearance.toLowerCase().equals("greg")){ //TESTING GREGORIAN WIDGET
+                            questionWidget = new GregorianWidget(context, fep);
+                        }else if(appearance != null && appearance.toLowerCase().equals("calendar")){
+                            questionWidget = new CalendarWidget(context, fep);
```

Comment:
Higher level question: how does CalendarWidget and GregorianWidget differ? Is it possible GregorianWidget should be a superclass of CalendarWidget?